OOP/Assignment1/Matrix.java

```diff
@@ -0,0 +1,71 @@
+class Matrix {
+
+	int data[][];
+	int noRows;
+	int noCols;
+
+	public Matrix(int row, int col) {
+
+		this.noCols = col;
+		this.noRows = row;
+		data = new int[row][col];
+
+	}
+
+	void addElements(int row, int col, int val) {
+		try {
+			data[row][col] = val;
+		} catch (ArrayIndexOutOfBoundsException e) {
+
+			System.out.println("wrong input");
+		}
+
+	}
+
+	public Matrix transpose() {
+
+		Matrix matrix = new Matrix(noRows, noCols);
+
+		for (int i = 0; i < noRows; i++) {
+			for (int j = 0; j < noCols; j++) {
+
+				matrix.data[i][j] = data[j][i];
+
+			}
+
+		}
+
+		return matrix;
+
+	}
+
+	public void show() {
+
+		for (int i = 0; i < noRows; i++) {
+			for (int j = 0; j < noCols; j++) {
```

Comment:
You could have iterated this loop till nolCols-2 so that 'System.out.print()' is used n-1 times and the 'System.out.println()' statement could print the (n-1)th element, in this way you could have avoided a blank println() statement. :)